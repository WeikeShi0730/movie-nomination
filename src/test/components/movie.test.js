import React from "react";
import { shallow } from "enzyme";
import Movie from "../../components/movie/movie.component";

it("should render Moive component", () => {
  const mockMovie = {
    Title: "Test Movie Name",
    Year: 1024,
  };
  expect(shallow(<Movie movie={mockMovie} />).getElement()).toMatchSnapshot();
});
