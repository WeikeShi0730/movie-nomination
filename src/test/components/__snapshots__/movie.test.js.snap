// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Moive component 1`] = `
<Context.Provider
  value={
    Object {
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "subscription": Subscription {
        "handleChangeWrapper": [Function],
        "listeners": Object {
          "notify": [Function],
        },
        "onStateChange": [Function],
        "parentSub": undefined,
        "store": Object {
          "clearActions": [Function],
          "dispatch": [Function],
          "getActions": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
        },
        "unsubscribe": null,
      },
    }
  }
>
  <Memo(Connect(Movie))
    movie={
      Object {
        "Title": "Test Movie Name",
        "Year": 1024,
      }
    }
  />
</Context.Provider>
`;
